mmeval/metrics/utils/nms.py

```diff
@@ -0,0 +1,250 @@
+# ------------------------------------------------------------------------------
+# Adapted from https://github.com/leoxiaobin/deep-high-resolution-net.pytorch
+# Original licence: Copyright (c) Microsoft, under the MIT License.
+# ------------------------------------------------------------------------------
+
```

Comment:
Maybe this file should be moved to `metrics/_vendor`? What do you think @zhouzaida